to_production.txt

```diff
@@ -1,15 +1,114 @@
 # New
+ofl/notoserifdivesakuru # https://github.com/google/fonts/pull/9047
+ofl/wdxllubrifonttc # https://github.com/google/fonts/pull/9380
```

Comment:
Will need a file size assessment on this one before the prod push.  @garretrieger have you seen any flags that this CJK familiy is too large for Workspace?

Reply:
The sandbox presubmit didn't raise any errors for size so I think it should be OK.